### PR TITLE
Feature/questionnaire reports

### PR DESCRIPTION
**Pull request by carmeltsadok**

### CODE REVIEW

File: CreateEventsReport/index.ts [hunk 1]
```diff
@@ -1,135 +1,122 @@
...
+
+        switch (user.role) {
+            case Role.Mada:
+            case Role.Itur:
+            case Role.Technical:
+                if (unitIds?.length) {
+                    filteredEvents = events.filter((event: any) => unitIds.includes(event.nodeData.unitId));
+                } else {
+                    filteredEvents = events;
+                }
```
Comment (lustigy96) [inline CreateEventsReport/index.ts:49]:
```suggestion
filteredEvents  = unitIds?.length? events.filter((event: any) => unitIds.includes(event.nodeData.unitId)) : events
```

Comment (lustigy96) [reply CreateEventsReport/index.ts:49]: I think that if unitIds = undefined, the if state returns true... (I think I had this bug) 
so maybe check that

File: CreateEventsReport/index.ts [hunk 2]
```diff
@@ -1,135 +1,122 @@
...
+            fields: [
+                config.fields.nodeName,
+                config.fields.nodeId,
+                config.fields.malshabId,
+                config.fields.malshabFirstName,
+                config.fields.malshabLastName,
+                config.fields.time,
+                config.fields.location,
+                config.fields.interviewersIds,
+                config.fields.results,
```
Comment (lustigy96) [inline CreateEventsReport/index.ts:110]: better to use a map, or jest get object values

File: CreateQuestionnaireReport/index.ts [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        const csv = JSONToCSV(dataToExport, {
+            fields: [
+                config.fields.nodeName,
+                config.fields.nodeId,
+                config.fields.malshabId,
+                config.fields.malshabFirstName,
+                config.fields.malshabLastName,
+                config.fields.eventTime,
+                config.fields.interviewersIds,
+                config.fields.notes,
```
Comment (lustigy96) [inline CreateQuestionnaireReport/index.ts:117]: get object's values and concat them...

**Review by lustigy96 — CHANGES_REQUESTED** [review]

File: CreateEventsReport/index.ts (hunk 1 above)
Comment (carmeltsadok) [reply CreateEventsReport/index.ts:49]: didn't return true

**Review by lustigy96 — DISMISSED** [review]

**Review by maykremer — APPROVED** [review]

**Pull request merged**